src/static/home.html

```diff
@@ -6,78 +6,84 @@
-                    <label for="signup" class="close-btn"></label>
-                    <div class="text">Sign Up</div>
-                    <form action="/singup" method="POST">
-                        <div class="data">
-                            <label for="signEmail">Email</label>
-                            <input id="signEmail" type="text" required>
-                        </div>
-                        <div class="data">
-                            <label for="verEmail">Verify Email</label>
-                            <input id="verEmail" type="text" required>
-                        </div>
-                        <div class="data">
-                            <label for="signPassword">Password</label>
-                            <input id="signPassword" type="password" required>
-                        </div>
-                        <div class="data">
-                            <label for="verPassword">Re-enter Password</label>
-                            <input id="verPassword" type="password" required>
-                        </div>
-                        <div class="btn">
-                            <div class="inner"></div>
-                            <button type="submit">Sign Up!</button>
-                        </div>
-                    </form>
+                <!-- HTML for login button/modal -->
+                <div class="login-head">
+                    <input type="checkbox" id="login">
+                    <label for="login" class="login-btn">Login</label>
+                    <div class="container">
+                        <label for="login" class="close-btn"></label>
+                        <div class="text">Login</div>
+                        <form action="/login" method="POST">
+                            <div class="data">
+                                <label for="email">Email</label>
+                                <input id="email" type="text" required>
+                            </div>
+                            <div class="data">
+                                <label for="password">Password</label>
+                                <input id="password" type="password" required>
+                            </div>
+                            <div class="forgot-pass"><a href="#">Forgot Password?</a></div>
+                            <div class="btn">
+                                <div class="inner"></div>
+                                <button type="submit">login</button>
+                            </div>
+                            <div class="signup-link">Not a member? <a href="/signup">Signup now</a></div>
+                        </form>
+                    </div>
                 </div>
             </div>
-
-            <!-- Sign up with Email button also opens up signup modal above -->
-            <div>
-                <label for="signup" class="signup-btn">Sign Up with Email!</label>
+        </div>
+        <main>
+            <div class="home-main">   
```

Comment:
div with class="home-main" could just use the main tag that already encompasses all of the same content

Reply:
Yep fixed that